composer.json

```diff
@@ -48,7 +48,11 @@
         "doctrine/orm": "The Doctrine ORM integration is optional in the bundle."
     },
     "autoload": {
-        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" }
+        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" },
+        "exclude-from-classmap": ["Tests/"]
+    },
+    "autoload-dev": {
+        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "Tests/" }
```

Comment:
this autoloading rule is useless, as it is broken (the prefix is wrong) and these classes are still autoloaded by the prod PSR-4 rule (would not be autoloaded in classmap-authoritative mode, but there is no reason to enable it during development)